### PR TITLE
chore: upgrade illuminate support

### PR DESCRIPTION
**Pull request by hisop**
Refs: PRO-777

Upgrade illuminate support

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,6 +16,8 @@
         }
     },
     "require-dev": {
+        "roave/security-advisories": "dev-latest"
+        ,
```
Comment (P-u-r-i-a) [inline composer.json:20]: 🍺 
```suggestion
        "roave/security-advisories": "dev-latest"
        ,
```
```suggestion
        "roave/security-advisories": "dev-latest"
        ,
```
```suggestion
        "roave/security-advisories": "dev-latest",
```

Comment (P-u-r-i-a) [inline composer.json:20]: 🍺 🤓 Out of curiosity why we need this dependency?
```suggestion
        "roave/security-advisories": "dev-latest",
```

Comment (hisop) [reply composer.json:20]: Its a safety check against dependancies with documented security issues, the PHP Security Advisories Database suggests it as best practice to be included. https://github.com/Roave/SecurityAdvisories https://phpqa.io/projects/roave-security-advisories.html#:~:text=Roave%20Security%20Advisories%20ensures%20that,the%20PHP%20Security%20Advisories%20Database.

Comment (P-u-r-i-a) [reply composer.json:20]: Awesome, Thanks for explaining that 🙏

**Review by usrNotFound — APPROVED** [review]
LGTM!!!

**Pull request merged**